shell.nix

```diff
@@ -0,0 +1,20 @@
+let
```

Comment:
If you'd like to use `nix build` itself -- I can help there.
Useful if you want others to install via Nix.

Reply:
/gcbrun